elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/SearchTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.elasticsearch;
```

Comment:
Why choose to create a new class for Search?

Reply:
The main purpose of 'AggregationTest' is for aggregation tests which focus on the aggregation logic. With this PR, 2 search test cases (8 tests) are added. Maybe it is a little bit confusing to still leave them in 'AggregationTest'  
Another reason is the test data of 'Search' is a little bit different from 'Aggregation' for covering more scenarios of 'Search to Range'. In order to avoid to break the existing test cases of 'Aggregation', so I created a separate test class for Search.

Reply:
Thanks. PR overall LGTM. I will take another pass soon.